include/revng/Model/VerifyHelper.h

```diff
@@ -144,9 +145,11 @@ class VerifyHelper {
   }
 
 public:
-  [[nodiscard]] bool isGlobalSymbol(const model::Identifier &Name) const;
-  [[nodiscard]] bool registerGlobalSymbol(const model::Identifier &Name,
-                                          const std::string &Path);
+  [[nodiscard]] bool isGlobalSymbol(std::string_view Name) const;
```

Comment:
I'd prefer we stick with `StringRef` all over the place, unless we globally decide to switch to `string_view` (and retain `StringRef` for LLVM API only).

Reply:
Done, see the new dedicated commit (`Replace std::string_view with llvm::StringRef`)

Note that I had to roll some back to `std::string_view` because `llvm::StringRef` lacks some constexpr capabilities (for example, you cannot even compare strings).

Reply:
Thanks, I know you love `string_view` but I really want to strive for coherency in the codebase.